src/test/java/io/dropwizard/kafka/integration/DropwizardKafkaIntegrationTest.java

```diff
@@ -35,11 +36,11 @@
 import java.util.concurrent.Future;
 import java.util.stream.Collectors;
 
-import javax.validation.Validator;
-
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class DropwizardKafkaIT {
+@SuppressWarnings("UnstableApiUsage")
+public class DropwizardKafkaIntegrationTest {
```

Comment:
Renaming to a more readable alternative.

Reply:
The class naming ending in `IT` is on purpose so that it is automatically processed as an integration test by the maven-failsafe-plugin. Changing this class name without updating the plugin's configuration will result in the class not being executed.